Reference/Management/Services/UserService/Create-a-new-user.md

```diff
@@ -0,0 +1,85 @@
+# Create a new user
+
+The document illustrates how you can create a new backoffice user from a Razor view. 
+
+
+### Getting a reference to the UserService
+The first step is to get a reference to the `UserService`. This is useful, as we're using in multiple times, and therefore doesn't have to reference it through `ApplicationContext.Current.Services.UserService` each time:
+
+    // Get a reference to the user service
+    IUserService us = ApplicationContext.Current.Services.UserService;
+    
+### Creating the user
+With the reference in place, we can now create the user:
+
+    // Creates a new user
+    IUser user = us.CreateUserWithIdentity("john-doe@xample.org", "john-doe@xample.org");
+    
+The `CreateUserWithIdentity` method takes two parameters - `username` and `email`. However in newer versions of Umbraco, it's considered best practice to also use the email address as the username. Calling `CreateUserWithIdentity` will create the user in the database, so it is not necessary to call the `Save` method for this (unless you further update the user as shown below).
+
+### Setting a password
+So far we have created the user, but we haven't yet set a password. We can do this through the `SavePassword` method, and Umbraco will then handle all the necessary password hashing for us:
+
+    // Set a new password for the user
+    us.SavePassword(user, "HelloWorld1234");
```

Comment:
This is a grey area (but i see you've noted some of this below)

This is not exactly the way to deal with passwords. Passwords policies are all governed by ASP.NET Identity and the user service does not wrap this (and cannot because there would be a circular reference). Currently this method wraps the membership provider that ASP.NET identity talks to but in recent versions of Umbraco (IIRC 7.7), this method is no longer used at all... it in fact is obsoleted and hidden but not at the IUserService interface, only the UserService class (oops!), so those attributes need to be added at the interface level too. 

This could actually cause some problems and the user won't be able to login since if the ASP.NET identity `IPasswordHasher` is replaced with a custom one.  The password hashed here will always be done with the membership provider but if the `IPasswordHasher` is replaced with a custom one

The other caveat to this is that it will work currently if the `IPasswordHasher` is not replaced by a custom one but only in a couple scenarios: 
* the user is new and doesn't have a password set
* during installation to set the admin password

Otherwise this method will fail because to change someone's password with the membership provider you have to pass in the current password and this method doesn't support that which is also why it's obsoleted and no longer used.

Remember, the services are for persisting data and talking to the database. 

So if you really want to create a new user, you'd use ASP.NET identity APIs like we do in core. However if you want to write information directly to the database, you'd use the user service. Both the SavePassword methods on IUserService and IMemberService are or should be obsolete. These methods existed a long time ago but should have really never been created and more importantly will no longer work in many cases since the membership providers  are updated to require an existing password to change them and with users ASP.NET Identity is in charge.